.github/workflows/release.yml

```
@@ -0,0 +1,191 @@
+          - minor
+          - major
+      custom_version:
+        description: 'Custom version (optional, overrides version_type)'
+        required: false
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)